### PR TITLE
Use interpolated strings in test code

### PR DESCRIPTION
**Pull request by halgab**
Contributes to #8203

## Test methodology

Static code analysis, CI

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/8720)

### CODE REVIEW

File: src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/RichTextBoxes.cs [hunk 1]
```diff
@@ -58,24 +58,32 @@ private unsafe void MakeLink(RichTextBox control, string text)
         private string ReportLinkClickedEventArgs(object sender, LinkClickedEventArgs e)
         {
             var control = (RichTextBox)sender;
-            var prefix = control.Text.Remove(e.LinkStart);
```
Comment (dreddy-work) [inline src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/RichTextBoxes.cs:61]: you may want to do this in a separate PR by enabling analyzers. Posible project by project.

Comment (halgab) [reply src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/RichTextBoxes.cs:61]: Okay, no problem. I just thought that as those values were only used in the interpolated string below that was in scope, but I can rollback if you prefer.
I don't know of any analyzer that can flag this though, so I guess this change will probably be lost for now

Comment (dreddy-work) [reply src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/RichTextBoxes.cs:61]: Understand. Wanted to group changes to be able to track easily in case any regression issues. I see [doc](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1831) that may help us enable analyzer in follow-up work.

Comment (halgab) [reply src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/RichTextBoxes.cs]: That's rollbacked. Thanks for the link. Looking at [this file](https://github.com/dotnet/winforms/blob/83f3a8476d682f8b74c7dce5505cc39560298447/eng/CodeAnalysis.test.globalconfig#L325) makes me think this analyzer is already enabled, but I'm not sure I'm looking at the right place...

Comment (dmitrii-drobotov) [reply src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/RichTextBoxes.cs]: @halgab - yes, that's the correct file, but it seems CA1831 is triggered only with array indexer operators, which is not the case here. There's also [CA1846](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1846), but that one is apparently only for method arguments. I didn't find other rules that would work in this case, so I guess we'll have to search for these cases manually.

**Review by dmitrii-drobotov — APPROVED** [review]
